fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -346,6 +346,23 @@ ACTOR Future<Void> getRate(UID myID,
 	}
 }
 
+void proxyGRVThresholdExceeded(const GetReadVersionRequest* req, GrvProxyStats* stats) {
+	++stats->txnRequestErrors;
+	req->reply.sendError(proxy_memory_limit_exceeded());
```

Comment:
This has an associated block that can be removed in NativeAPI:

https://github.com/apple/foundationdb/blob/6c2165dfeef4961cb17f5282c36573797f92020b/fdbclient/NativeAPI.actor.cpp#L4979

The NativeAPI change should probably only be made on master. The change here should be ok to port back to release-7.0, and neither change should go to release-6.3.

I'm double checking if there are any other concerns with this.

Reply:
Done.